client/src/colorPicker/colorPicker.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from "react";
+import { SketchPicker } from "react-color";
+
+interface Props {}

```

Comment:
Can you try adding a callback here to see if the color is passed back upon a user's choice? Or do you plan to do that in another PR?